documentation/airport-guides/kiad.md

```diff
@@ -0,0 +1,151 @@
+Potomac Approach (LUCKE Sector)
+Potomac Approach (MANNE Sector)
+Potomac Approach (MULRR Sector)
+Potomac Approach (RCOLA Sector)
+```
+
+### Restricted Airspace
+With Dulles' close proximity to downtown Washington, D.C., it is important to be cognizant of the various restricted airspace in the region. The areas around the National Mall and US Naval Observatory are both prohibited airspace with a ceiling of 18,000ft.
+
+For details of other restricted airspaces, see <a  href="https://www.faa.gov/documentLibrary/media/Order/FAA_Order_JO_7400.8Z_Special_Use_Airspace.pdf" target="_blank">FAA Order JO 7400.8Z</a>
+
+##### P-56A, P-56B
+P-56A is the airspace which seeks to protect the National Mall and various other monuments in Washington D.C.
+
+P-56B is a circle of 0.5nm radius around the U.S. Naval Observatory.
+
+Designated altitudes. Surface to 18,000 feet MSL.
+Time of designation. Continuous.
+Using agency. United States Secret Service, Washington, DC.
+
+##### Hill Military Operations Area
+Hill MOA is in the south of the Mount Vernon Area.
+
+Altitudes. Surface to and including 3,000 feet MSL.
+Times of use. Sunrise to sunset.
+Controlling agency. FAA, Potomac Approach.
+Using Agency. U.S. Army, Fort A. P. Hill, Bowling Green, VA.
+
+##### Demo 1 Military Operations Area
+The Demo 1 MOA is in the southwest of the Mount Vernon Area.
+
+Altitudes. 500 feet to 5,000 feet MSL.
+Times of use. Intermittent by NOTAM.
+Controlling agency. FAA, Potomac TRACON.
+Using agency. U.S. Marine Corps, Commanding General, Marine Corps Base, Quantico, VA.
+
+### Other airports within the Shenandoah Area
+##### KFDK/FDK - Frederick Municipal Airport
+Frederick Municipal Airport is classified as a general aviation airport. According to analysis, FDK experienced approximately 129,000 operations in 2004 with an expected increase to about 165,000 by 2025.
+
+|Runway|Length (ft)|Surface|
+|----|----|----|
+|5/23|5,219|Asphalt concrete|
+|12/30|3,600|Asphalt|
+
+##### KHEF/HEF - Manassas Regional Airport
+Manassas Regional Airport is five miles southwest of the center of Manassas, VA. It is also the largest regional airport in Virginia, and it is located 30 miles (48 km) from Washington, D.C..
+
+|Runway|Length (ft)|Surface|
+|----|----|----|
+|16L/34R|6,200|Asphalt|
+|16R/34L|3,704|Asphalt|
+ 
+### Videomap
+The videomap contains the following:
+
+* Restricted Airspaces;
+* Extended centerlines for all runways (1L, 1C, 1R, 12, 19L, 19C, 19R, 30);
+* Various airports within in and around the Shenandoah Area;
+* Locations of important fixes;
+* The Potomac River
+
+The map is courtesy of VATSIM's vZDC vSTARS file.
+
+### Additional Resources
+<a  href="https://i.ibb.co/bQSXRmW/image.png" target="_blank">SHD Airspace Diagram</a>
+<a  href="https://www.vzdc.org/storage/files/SHD%20Arrival%20Flow%20Chart_1585847150.png" target="_blank">IAD Arrival Flows Diagram</a>
+<a  href="https://vzdc.org/downloads/PCT_SOP.pdf" target="_blank">vZDC PCT SOP</a>
```

Comment:
This link doesn't seem to work anymore

Reply:
I think you're referring to the PCT SOP? I fixed it for DCA but forgot to fix it here 🤦‍♂️